Terminal.Gui/Views/ComboBox.cs

```diff
@@ -81,7 +81,7 @@ private void Initialize()
 			};
 
 			// TODO: LayoutComplete event breaks cursor up/down. Revert to Application.Loaded 
```

Comment:
I just noticed this comment. What about `LayoutComplete` breaks cursor up/down? Have you submitted a bug against that? It shouldn't.